app/src/main/java/app/revanced/twitch/settingsmenu/SettingsHooks.java

```diff
@@ -0,0 +1,110 @@
+package app.revanced.twitch.settingsmenu;
+
+import static app.revanced.twitch.utils.ReVancedUtils.getIdentifier;
+import static app.revanced.twitch.utils.ReVancedUtils.getStringId;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import app.revanced.twitch.utils.ReVancedUtils;
+import app.revanced.twitch.utils.LogHelper;
+import tv.twitch.android.feature.settings.menu.SettingsMenuGroup;
+import tv.twitch.android.settings.SettingsActivity;
+
+public class SettingsHooks {
+    private static final int REVANCED_SETTINGS_MENU_ITEM_ID = 0x7;
+    private static final String EXTRA_REVANCED_SETTINGS = "app.revanced.twitch.settings";
+
+    /**
+     * Launches SettingsActivity and show revanced settings
+     */
+    public static void startSettingsActivity() {
+        LogHelper.debug("Launching revanced settings");
+
+        ReVancedUtils.ifContextAttached((c) -> {
+            Intent intent = new Intent(c, SettingsActivity.class);
+            Bundle bundle = new Bundle();
+            bundle.putBoolean(EXTRA_REVANCED_SETTINGS, true);
+            intent.putExtras(bundle);
+            intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+            c.startActivity(intent);
+        });
+    }
+
+    /**
+     * Helper for easy access in smali
+     * @return Returns string resource id
+     */
+    public static int getRevancedSettingsString() {
```

Comment:
Probably want to use the casing of `ReVanced` here.

Reply:
Fixed with 2665f5a